distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -115,7 +115,7 @@ grant codeBase "file:/registry-federation-admin-impl" {
     permission java.io.FilePermission "${ddf.home.perm}etc${/}registry${/}-", "read";
 }
 
-grant codeBase "file:/spatial-csw-source/spatial-csw-endpoint/org.apache.camel.camel-core/com.google.guava/ddf-pubsub/registry-source-configuration-handler/registry-api-impl/org.apache.camel.camel-blueprint" {
+grant codeBase "file:/spatial-csw-source/spatial-csw-endpoint/org.apache.camel.camel-core/com.google.guava/ddf-pubsub/registry-source-configuration-handler/registry-api-impl/org.apache.camel.camel-blueprint/fabric8-karaf-checks" {
```

Comment:
❓ Do we really want to give read access to the keystores to fabric8?

Reply:
I had to in order to get the itests to actually pass, they were running into access violations and the bundle wasn't starting because of that

Reply:
That means we are exposing this in production to address an itests problem. 
@stustison What is your take on that?

Reply:
I'm not sure why I don't run into this problem when running the actual system and granting only the `createClassLoader` permission to the health check bundle. Everything seems to be working for me outside of itests in that case...

Reply:
I definitely don't like it. It might be time to explore having an itest permission file that we can drop in just for the tests when stuff like this happens.

Reply:
I can go with that approach if that will be preferred @stustison. Would we want it to be inherited by downstream projects automatically?

Reply:
Yeah so did some more testing and these are definitely required permissions for this feature to work